ote_cli/ote_cli/utils/io.py

```diff
@@ -236,3 +242,69 @@ def add_subtask_labels(dataset, info):
         return label_schema
 
     return LabelSchemaEntity.from_labels(dataset.get_labels())
+
+
+def get_image_files(root_dir):
+    """
+    recursively get all image file paths from given root_dir
+    """
+    img_data_formats = [
+        ".jpg",
+        ".jpeg",
+        ".gif",
+        ".bmp",
+        ".tif",
+        ".tiff",
+        ".png",
+    ]
+    img_data_formats.extend([x.upper() for x in img_data_formats])
+    img_data_formats = tuple(img_data_formats)
+
+    # single image path
+    if root_dir.endswith(img_data_formats):
+        return [os.path.dirname(root_dir), os.path.basename(root_dir)]
+
+    img_files = []
+    for root, _, _ in os.walk(root_dir):
+        for format_ in img_data_formats:
+            img_files.extend(
+                [(root, file.name) for file in Path(root).glob(f"*{format_}")]
+            )
+    return img_files if img_files else None
+
+
+def save_saliency_output(
```

Comment:
Is there any requirements that describe of how the saliancy map has to looks like? Please make sure that this post-processing is correct and post the examples of the output `saliency_maps` in the PR discussion.

Reply:
Yes, there were some change in extracting the saliency map. So I'll re-test it. Let's see the result together

Reply:
There were some logic err in extracting the `saliency_map`, and fixed in [06ff2da](https://github.com/openvinotoolkit/training_extensions/pull/1311/commits/06ff2dac6254ef1b519a9a85217269ef821fde7e). Thank you for investigation. Test images will be added in #1334.

![image](https://user-images.githubusercontent.com/49050419/200238359-3a2ac9e8-dd59-454d-8f01-ac12fadc4759.png)